numpy/_core/src/umath/loops_hyperbolic.dispatch.cpp.src

```diff
@@ -5,12 +5,18 @@
  ** neon_vfpv4
  ** vx vxe
  **/
+
 #include "numpy/npy_math.h"
 #include "simd/simd.h"
 #include "loops_utils.h"
 #include "loops.h"
+#include "fast_loop_macros.h"
+
+
+#if NPY_SIMD_FMA3 && (NPY_SIMD_F32 || NPY_SIMD_F64) // requires native FMA
```

Comment:
Highway always has MulAdd, but we should indeed check `#if HWY_HAVE_FLOAT64` here. That is only false on Armv7. We can skip the previous checks, I think.